guide/onboarding/automatic-cloud-backup.md

```diff
@@ -9,19 +9,21 @@ permalink: /guide/onboarding/backing-up-a-recovery-phrase/automatic-cloud-backup
 main_classes: -no-top-padding
 ---
 
-# Automatic cloud backup
+# Cloud backup
 
 ### How it works
 
-The user’s key is encrypted and backed up on their respective device/cloud storage provider. If their device is lost, stolen, or broken, the user can log into their cloud storage account (such as Apple iCloud or Google Drive) to regain access to their funds. To learn more about the technical details, hop over to the [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section on this topic.
+The user's wallet recovery data and channel state are backed up on their respective cloud storage provider. The recovery phrase only needs to be backed up once. However, the channel state may need to be backed up frequently because it will change every time the user's wallet sends, receives, or routes a payment; therefore, the channel state should be backed up automatically.
```

Comment:
```suggestion
The user's wallet recovery data and channel states are backed up on their respective cloud storage provider. The recovery phrase only needs to be backed up once. However, channel states may need to be backed up frequently because they change every time the user's wallet sends, receives, or routes a payment; therefore, the channel state should be backed up automatically.
```